session/manager_test.go

```diff
@@ -0,0 +1,23 @@
+package session
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestManagerHasSessionsStored(t *testing.T) {
```

Comment:
Does not look like unit.
Is it like integration test?

Reply:
Yeah, testing the manager is kind of pointless in this case.. the only use case for unit tests would be to freeze the interface methods, does that make sense though?